src/main/java/com/google/sps/model/ActionableMessageHelper.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.model;
+
+import com.google.api.services.gmail.model.Message;
+
+/** Contains methods that help create the non-trivial fields in an ActionableMessage */
+public interface ActionableMessageHelper {
+  /**
+   * Priority is determined by a series of arbitrary factors. If a message is starred by a user, it
+   * is considered high priority. Then, if a message is not from a mailing list or marked important,
+   * it is marked medium priority Otherwise, the message is of low priority. higher priority than a
+   * message where both of these are false. Finally, if a message was sent later than another
+   * message, it is considered to be of higher priority.
```

Comment:
"Priority is determined by a series of arbitrary factors. If a message is starred by a user, it is considered high priority. Then, if a message is not from a mailing list or marked important, it is marked medium priority Otherwise, the message is of low priority."

nit: I think the doc could stop there

Reply:
Agreed - thanks!